kolibri/plugins/style_guide/assets/src/views/content/navbar/index.vue

```diff
@@ -1,31 +1,57 @@
 <template>
 
-  <page-template title="Horizontal Navbar" :completed="false">
+  <page-template :completed="true">
+
+    <h1>Horizontal Navbar</h1>
+
+    <p>
+      The horizontal navbar should be the primary means of navigating between
+      sub-pages of top-level sections (app plugins) of Kolibri.
+    </p>
+
+    <p>
+      Navbar links use information from vue router to determine which one is
+      visually selected.
+    </p>
+
+    <show>
+      <k-navbar>
+        <k-navbar-link
+          type="icon-and-title"
+          title="Trending"
+          icon="trending_up"
+          :link="a">
+        </k-navbar-link>
+        <k-navbar-link
+          type="icon-and-title"
+          title="Lit"
+          icon="whatshot"
+          :link="b">
+        </k-navbar-link>
+        <k-navbar-link
+          type="icon-and-title"
+          title="Favorites"
+          icon="favorite"
+          :link="c">
+        </k-navbar-link>
+      </k-navbar>
+    </show>
+
+    <h2>Usage guidelines</h2>
+
+    <p>Text for navbar items should ideally be a single, short word.</p>
+
     <h2>Component APIs</h2>
-    <h3><code>{{ kNavbarApi.name }}</code></h3>
+
+    <h3>Navbar</h3>
     <component-docs :api="kNavbarApi" />
```

Comment:
Would be good to update the component description of the kNavbar Currently: `Used for navigation within a plugin`